docs/recipes.md

```diff
@@ -105,6 +105,18 @@ In your leak reporting code:
 val retainedInstanceCount = AppWatcher.objectWatcher.retainedObjectCount
 ```
 
+## LeakCanary in release builds
+
+We **don't recommend** including LeakCanary in your production build. So to avoid that you accidentally include the `com.squareup.leakcanary:leakcanary-android` dependency in a release build, LeakCanary crashes during initialization if the APK is not debuggable.
```

Comment:
Nit: since we're putting emphasize on these words, I think it would be better to spell it out - **do not recommend** - instead of shortening.